gips/0000-template.md

```diff
@@ -1,10 +1,9 @@
 ---
-GIP: <The number of this GIP. To be assigned by editors.>
+GIP: <The number of this GIP. Selected by the author and confirmed by editors (which may change numbers to resolve conflicts)>
 Title: <The title of this GIP>
 Authors: <A list of authors. Include full name or pseudonym. At least one author must have valid contact information provided in angle brackets.>
 Created: <The date the GIP was created.>
-Updated: <The date the GIP was last updated.>
-Stage: <The current stage of the proposal. Specified by the author and confirmed by editors by virtue of a GIP being accepted into an editor's view of the repo.>
+Stage: <The current stage of the proposal. Specified by the author and confirmed by editors by virtue of a GIP being accepted into the main branch of the GitHub repo.>
```

Comment:
Because this is a template, would it be best to reference GIP-0001 for authors to know what the official stages are?